<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>550</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>762</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GENEL BARKOD SİSTEMİ</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
  
	font: 700 9pt &quot;Microsoft PhagsPa&quot;;

}

QFrame {
	
	background-color: rgb(51, 51, 49)
}

QHeaderView {
 	background-color: transparent;
}


QFrame#frame {
 	 border-radius: 30px;
}

QPushButton {
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_16">
         <property name="styleSheet">
          <string notr="true">background-color: none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="topMargin">
           <number>7</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_12">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QFrame" name="frame_14">
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item alignment="Qt::AlignmentFlag::AlignHCenter">
              <widget class="QLabel" name="label">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../resources/resource.qrc">:/main/db.png</pixmap>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_13">
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item alignment="Qt::AlignmentFlag::AlignHCenter">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Genel Barkod Sistemi</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="maximumSize">
             <size>
              <width>666</width>
              <height>437</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>15</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_4">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(228, 237, 202);
border-radius: 10px;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QTableWidget" name="tableWidget">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777214</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-radius: 30px;</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::NoFrame</enum>
                  </property>
                  <property name="verticalScrollBarPolicy">
                   <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
                  </property>
                  <property name="horizontalScrollBarPolicy">
                   <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                  </property>
                  <property name="selectionBehavior">
                   <enum>QAbstractItemView::SelectionBehavior::SelectColumns</enum>
                  </property>
                  <property name="showGrid">
                   <bool>true</bool>
                  </property>
                  <property name="rowCount">
                   <number>40</number>
                  </property>
                  <property name="columnCount">
                   <number>5</number>
                  </property>
                  <attribute name="horizontalHeaderVisible">
                   <bool>true</bool>
                  </attribute>
                  <attribute name="horizontalHeaderHighlightSections">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderCascadingSectionResizes">
                   <bool>true</bool>
                  </attribute>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <row/>
                  <column/>
                  <column/>
                  <column/>
                  <column/>
                  <column/>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_5">
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QFrame" name="frame_6">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>630</width>
                    <height>204</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Shadow::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QFrame" name="frame_8">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>303</width>
                       <height>186</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::Shape::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Shadow::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_5">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="sizeConstraint">
                       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
                      </property>
                      <item alignment="Qt::AlignmentFlag::AlignHCenter">
                       <widget class="QFrame" name="frame_9">
                        <property name="frameShape">
                         <enum>QFrame::Shape::NoFrame</enum>
                        </property>
                        <property name="frameShadow">
                         <enum>QFrame::Shadow::Raised</enum>
                        </property>
                        <layout class="QHBoxLayout" name="horizontalLayout_6">
                         <item>
                          <widget class="QFrame" name="frame_11">
                           <property name="frameShape">
                            <enum>QFrame::Shape::NoFrame</enum>
                           </property>
                           <property name="frameShadow">
                            <enum>QFrame::Shadow::Raised</enum>
                           </property>
                           <layout class="QVBoxLayout" name="verticalLayout_5">
                            <item>
                             <widget class="QPushButton" name="save_btn">
                              <property name="text">
                               <string>Kaydet</string>
                              </property>
                             </widget>
                            </item>
                            <item>
                             <widget class="QPushButton" name="pushButton_2">
                              <property name="text">
                               <string>Temizle</string>
                              </property>
                             </widget>
                            </item>
                           </layout>
                          </widget>
                         </item>
                        </layout>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item alignment="Qt::AlignmentFlag::AlignRight">
                    <widget class="QFrame" name="frame_7">
                     <property name="frameShape">
                      <enum>QFrame::Shape::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Shadow::Raised</enum>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_7">
                      <item>
                       <widget class="QLabel" name="label_3">
                        <property name="text">
                         <string>Öncelikle barkodları seçmelisiniz</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_4">
                        <property name="text">
                         <string>Ardından yazıcıyı seçmelisiniz seçtikten sonra seç butonuna tıklamalısınız</string>
                        </property>
                        <property name="scaledContents">
                         <bool>true</bool>
                        </property>
                        <property name="wordWrap">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_7">
                        <property name="text">
                         <string>Eğer ki belgeyi kaydetmek istiyorsanız kaydet'e tıklayınız (masaüstüne otomatik oluşturulacaktır)</string>
                        </property>
                        <property name="wordWrap">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_5">
                        <property name="text">
                         <string>Yazdır butonuna tıkladıktan sonra yazdırılacaktır</string>
                        </property>
                        <property name="wordWrap">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_6">
                        <property name="text">
                         <string>Temizle butonuna tıkladıktan sonra seçilen barkodlar temizlenecektir :)</string>
                        </property>
                        <property name="wordWrap">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
